packages/app/src/specs/banners/CreateOrganizationBanner.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <TrackedBanner
+    v-if="cohortOption"
```

Comment:
Without this check it's possible we render the banner and then pieces of text content pop in once the cohort is determined.